### DISCUSSION TITLE
Prefix events from spawned/invoked services?

### DISCUSSION THREAD

**Discussion by geekox86** [statelyai/xstate#2362] [Q&A] [answered]
Hello.
Is it possible to prefix the events sent from spawned/invoked services?

I want to keep event names short inside these services and only prefix them when usee inside parent service.

**Comment by davidkpiano** [comment, marked as answer]
You currently can't do this. What would an ideal API look like to you for this?

**Reply by geekox86** [reply]
I think a prefixEvents parameter passed to spawn would be great as long we can later identify the origin service of the event in parent service.
